index.d.ts

```diff
@@ -1,5 +1 @@
-import Sentry from "react-native-sentry";
-
-export default class SentryExpo extends Sentry {
-  static enableInExpoDevelopment: boolean;
```

Comment:
I get a type error since this is missing

![image](https://user-images.githubusercontent.com/1404810/64699385-f1ebd380-d4a4-11e9-9bf3-c0a2d2ea1091.png)

Reply:
fixed in v2.0.1